src/lib/components/Button/Button.tsx

```diff
@@ -124,6 +143,13 @@ const ButtonComponent = forwardRef<HTMLButtonElement | HTMLAnchorElement, Button
           )}
         >
           <>
+            {isProcessing && (
+              <Spinner
+                size={processing.spinnerSize}
+                light={processing.spinnerLight}
+                className={processing.spinnerClassName}
```

Comment:
This is a little bit tricky. We can't actually do this kind of thing... We need to use the `theme={}` props. So, basically, `FlowbiteButtonTheme` must have a `spinner` property that will use the `FlowbiteSpinnerTheme` interface type.